### ISSUE TITLE
Training with custom dataset gives purple images

### ISSUE THREAD

**Issue by Trainmaster9977** [NVIDIA/pix2pixHD#291]
Hello! I was training with my own dataset (RGB instead of with labels, and with a .5 learning rate, as the relationships between the data are slightly abstract) and left it going overnight. When I checked it in the morning, all the checkpoint results were fully purple images. I ran it for 18 epochs, and while thats a small number, this is definitely abnormal. It should still give me something at this number, even if it's not accurate in the slightest. Does anybody know why this might have turned out this way?
